### DISCUSSION TITLE
Toggle TCP/UDP/uTP usage independently possible?

### DISCUSSION THREAD

**Discussion by pataquets** [anacrolix/torrent#690] [Q&A] [answered]
Can TCP/UDP/uTP usage for piece transmission be individually toggled on/off?
Use case: being able to use UDP/uTP only (no TCP) to ease load on uplink equipment, at the expense of speed.
Thanks.

**Comment by anacrolix** [comment, marked as answer]
Yes, use `DisableTCP` in [`ClientConfig`](https://pkg.go.dev/github.com/anacrolix/torrent#ClientConfig).

**Comment by anacrolix** [comment]
Regarding https://github.com/transmission/transmission/issues/1519, the advice you have there is solid. There is still overhead to uTP, the reduced congestion effects are good for other forms of traffic but it's not perfect. Your best bet for bad hardware is definitely to reduce total connection count.
